Documentation/README.md

```diff
@@ -35,17 +35,66 @@
 
 - Developer Documentation
   - [API usage](./sources/api-usage.md#code-based-experiences-and-content-cards-apis)
-  - Public classes
-    - [ContentCard - DEPRECATED](./sources/propositions/content-card.md)
-    - [ContentType](./sources/propositions/schemas/content-type.md)
-    - [Proposition](./sources/propositions/proposition.md)
-    - [PropositionItem](./sources/propositions/proposition-item.md)
-    - [Surface](./sources/propositions/surface.md)
-    - Schema classes
-      - [ContentCardSchemaData](./sources/propositions/schemas/content-card-schema-data.md)
-      - [HtmlContentSchemaData](./sources/propositions/schemas/html-content-schema-data.md)
-      - [InAppSchemaData](./sources/propositions/schemas/inapp-schema-data.md)
-      - [JsonContentSchemaData](./sources/propositions/schemas/json-content-schema-data.md)
+- Public classes
+  - [ContentCard - DEPRECATED](./sources/propositions/content-card.md)
+  - [ContentType](./sources/propositions/schemas/content-type.md)
+  - [Proposition](./sources/propositions/proposition.md)
+  - [PropositionItem](./sources/propositions/proposition-item.md)
+  - [Surface](./sources/propositions/surface.md)
+- Schema classes
+  - [ContentCardSchemaData](./sources/propositions/schemas/content-card-schema-data.md)
+  - [HtmlContentSchemaData](./sources/propositions/schemas/html-content-schema-data.md)
+  - [InAppSchemaData](./sources/propositions/schemas/inapp-schema-data.md)
+  - [JsonContentSchemaData](./sources/propositions/schemas/json-content-schema-data.md)
+
+### Content cards with UI
+
+- Developer documentation 
+
+    - [API Usage](./sources/content-card-ui/api-usage.md)
+- Public Classes, Enums, and Interfaces
+
+    - [AepUI](./sources/content-card-ui/public-classes/aepui.md)
```

Comment:
Wondering if we should document the sealed interface here since customer cannot provide their own implementation. Do you think we could have a section "AepUI implementations" and document the implemented types like `SmallImageUI`?

Reply:
Actually nevermind, lets keep it. Looks like most other projects do. Can we make sure to add `SmallImageUI` in sub section under `AepUI`?

Reply:
i've removed `SmallImageUI`'s individual page and added it in the `AepUI` documentation